pkg/yxml/Makefile.yxml

```diff
@@ -0,0 +1,5 @@
+SRCS := yxml.c
```

Comment:
I still think it's not needed to use the SRCS variable: there's only one C file and the automatic C file selection should take it into account anyway.

Reply:
right. fixed.